packages/react-native-media-player/ios/MediaPlayer.swift

```diff
@@ -0,0 +1,25 @@
+@objc(MediaPlayer)
+public class MediaPlayer: NSObject {
+
+  @objc(load:resolver:rejecter:)
+  public func load(to url: String, resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) {
+    // Date is ready to use!
+    resolve(NSNull());
```

Comment:
Codacy found an issue: [Lines should not have trailing semicolons.](https://app.codacy.com/manual/yajananrao/Nadha/pullRequest?prid=5728824)